floor_annotation/lib/src/type_converters.dart

```diff
@@ -22,7 +22,7 @@ import 'package:meta/meta.dart';
 @experimental
 class TypeConverters {
   /// The list of type converter classes.
-  final List<Type> value;
+  final List<Type>? value;
```

Comment:
Why is this nullable? I have the feeling it makes not much sense to use this annotation if you do not have any type converters. In other words, the list will always have at least one element and will never need to be null